packages/SwingSet/src/kernel/vatAdmin/vatAdmin-src.js

```diff
@@ -34,8 +34,8 @@ export function buildRootDeviceNode({ endowments }) {
       const vatID = kernelVatCreationFn({ bundleName }, options);
       return vatID;
     },
-    terminate(_vatID) {
-      // TODO(hibbert)
+    terminate(vatID) {
+      kernelTerminateVatFn(vatID);
```

Comment:
already-terminated flag check needs to go here, before calling `kernelTerminateVatFn`